scripts/presubmit-tests.sh

```diff
@@ -317,8 +317,14 @@ function main() {
   fi
 
   run_build_tests || failed=1
-  run_unit_tests || failed=1
-  run_integration_tests || failed=1
+  # Don't run unit tests if build tests failed
+  if (( ! failed )); then
+    run_unit_tests || failed=1
```

Comment:
You're changing the behavior or the presubmit script, not the release script. The issue was about having this behavior only for releases. I suggest using a global environment variable to make the behavior configurable.

Reply:
Updated to use the PRESUBMIT_TEST_FAIL_FAST global environment variable to configure the presubmit-test.

Reply:
Updated to initialize the readonly env var at the beginning of the script.